src/plugins/discover/public/application/angular/doc_table/components/pager/tool_bar_pager_buttons.tsx

```diff
@@ -18,35 +19,41 @@ interface Props {
 
 export function ToolBarPagerButtons(props: Props) {
   return (
-    <div className="kuiButtonGroup">
-      <button
-        className="kuiButton kuiButton--basic kuiButton--icon"
-        onClick={() => props.onPagePrevious()}
-        disabled={!props.hasPreviousPage}
-        data-test-subj="btnPrevPage"
-        aria-label={i18n.translate(
-          'discover.docTable.pager.toolbarPagerButtons.previousButtonAriaLabel',
-          {
-            defaultMessage: 'Previous page in table',
-          }
-        )}
-      >
-        <span className="kuiButton__icon kuiIcon fa-chevron-left" />
-      </button>
-      <button
-        className="kuiButton kuiButton--basic kuiButton--icon"
-        onClick={() => props.onPageNext()}
-        disabled={!props.hasNextPage}
-        data-test-subj="btnNextPage"
-        aria-label={i18n.translate(
-          'discover.docTable.pager.toolbarPagerButtons.nextButtonAriaLabel',
-          {
-            defaultMessage: 'Next page in table',
-          }
-        )}
-      >
-        <span className="kuiButton__icon kuiIcon fa-chevron-right" />
-      </button>
-    </div>
+    <EuiFlexGroup responsive={false} gutterSize="s" alignItems="center">
+      <EuiFlexItem grow={false}>
+        <EuiButtonIcon
+          display="base"
+          iconType="arrowLeft"
+          iconSize="l"
+          size="s"
+          onClick={() => props.onPagePrevious()}
+          isDisabled={!props.hasPreviousPage}
+          data-test-subj="btnPrevPage"
+          aria-label={i18n.translate(
```

Comment:
@andreadelrio quick question, wouldn't smaller button with icons look better here? like this? 

<img width="1262" alt="Bildschirmfoto 2021-07-05 um 11 38 57" src="https://user-images.githubusercontent.com/463851/124451565-d17dbf00-dd85-11eb-8394-4d69ba91047d.png">

Reply:
changed to `iconSize="m"`, `size="xs"` in this case

Reply:
Would it be better to replace this buttons with EuiPaginate?

Reply:
In this case I'd say no, because the text displayed here is the number of docs , not the number of pages. Something that's missing when using the new table. Should find a nice solution for this later on

Reply:
> @andreadelrio quick question, wouldn't smaller button with icons look better here? like this?
> 
> <img alt="Bildschirmfoto 2021-07-05 um 11 38 57" width="1262" src="https://user-images.githubusercontent.com/463851/124451565-d17dbf00-dd85-11eb-8394-4d69ba91047d.png">

@kertal Yes, this looks good! cc @Dmitriynj

Reply:
Just wanted to add that I do think we should consider making the switch the `EuiPaginate` eventually, with a limited number of `rows per page` so that the table is more compact. I think that would fit more nicely in a Dashboard. We could switch from numbers of docs to showing number of pages,  and add a text indicating the number of # of documents if needed.

Reply:
if the number of document is displayed, this would be nice, however, could also be done in this herculean PR #104361

Reply:
@andreadelrio you can see updated pagination **[here](https://github.com/elastic/kibana/pull/104361#issue-683959498)**